### ISSUE TITLE
Feature Request: Please add Mail Merge to Nylas Mail

### ISSUE THREAD

**Issue by meritweb** [nylas/nylas-mail#3380]
Hello,

The Mail Merge/Mass Email feature is one that I use quite often, and it is very helpful.  I would like to see it added back into Nylas Mail.

**Comment by tongnk** [comment]
+1 to that. Not sure why the upgrade has meant we don't have it anymore...

**Comment by nkwsy** [comment]
Agreed, I don't know what to do without it. It was an essential part of my email client.

**Comment by JonasMS** [comment]
Fairly peeved. Just paid for Nylas pro because the `Mail Merge` feature was advertised as a pro feature.

Anyone know of a mail client that has a `Mail Merge` feature?

**Comment by agentlewis** [comment]
whats going on??? I had this feature last year and now its gone...

**Comment by agentlewis** [comment]
I found the release notes - its pretty clear whats going on. this issue should be closed. 

Nylas Mail 2.0 Release FAQ https://support.nylas.com/hc/en-us/articles/115005574208-Nylas-Mail-2-0-Release-FAQ

@meritweb I'd suggest you edit your first comment with a link to the release notes so others easily find them.

**Comment by meritweb** [comment]
@agentlewis This issue should not be closed, as it is a request to reinstate a feature, and was opened due to the following text in the release notes: " We are working to adding many of these features back to Nylas Mail, with a focus on the most requested features."  How will they know that mass email is a requested feature if no one requests it?

**Comment by agentlewis** [comment]
@meritweb right fair enough regarding closing. I only just twigged that Nylas Mail and N1 are technically different things. This is not yet a feature in Nylas Mail but was in N1 > ergo its a feature from the point of view of Nylas mail as you have stated. So sorry for that 🍑 

Can you then update the first post with a link to the release notes - people are unclear about whats going on and your post will be the first they skim read.

**Comment by JonasMS** [comment]
Thanks @bherila.

I've resorted to spending $25/mo for a CRM tool that does the job fairly seamlessly. It's funny how much people are willing to pay for a feature that saves several hours / month.

**Comment by bherila** [comment]
If it saves you 1 hour and you cost more than $25 an hour, then it is worth it :)

**Comment by s2imon** [comment]
Frankly, I am somewhat confused about what's going on exactly at Nylas. Nylas Pro/N1 was constantly buggy and stopped working for me last week. The team practically never answers tweets and I'm not sure which products have been abandoned for what reason or which live on. 

Is Nylas Pro even a thing now? If it is where can I re-download? I've had to install Nylas Mail to get the emails to work again, but I am missing the Mail Merge feature.

**Comment by nkwsy** [comment]
I just cant believe they have not gotten this done yet. I still was charged $80 for the pro and now I cant even use this garbage. Why on earth did you take something that was working and just break the hell out of it? 

I had a lot of hope for this project.
